torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -317,8 +317,8 @@ def _post_forward_reshard(
 def _root_pre_forward(
     state: _FSDPState,
     module: nn.Module,
-    *args,
-    **kwargs,
+    args,
+    kwargs,
```

Comment:
nit: types? (no need to update this PR)

Reply:
I am actually not exactly sure what the types would be.

For `kwargs`, would it be `Dict[str, Any]`? For `args`, would it be `Iterable[Any]`?

(I am guessing here that an object is an `Iterable` iff it can be unpacked with the `*` operator.)